README.md

```diff
@@ -1 +1,3 @@
 # hw01
+
+My name is Angel Chacon. 
```

Comment:
Hey dude, I'm just going to put all of my comments here since there aren't a ton of places to put them on this project.  I think you misunderstood the instructional video - the professor only put his name because he didn't need to show us the entire process.  There was more to the assignment: we were supposed to add a header to the file, bold/italicize something, add a picture, add a link, and add a list.  I'm not seeing any of that on any of your commits, so I can only assume you didn't see the homework overview.